lecture10/storage.yml

```
@@ -0,0 +1,25 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Resources: 
+  S3Bucket:
+    Type: AWS::S3::Bucket
+    Properties: 
+      Tags:
+        - Key: Name
+          Value: S3Bucket-CF
+  VPCEndpoint:
+    Type: AWS::EC2::VPCEndpoint
+    Properties:
+      PolicyDocument: 
+        Statement:
+          - Effect: Allow
+            Principal: "*"
+            Action: "s3:*"
+            Resource: 
+              - !Sub arn:aws:s3:::${S3Bucket}
+              - !Sub arn:aws:s3:::${S3Bucket}/*
+      RouteTableIds:
+        - !ImportValue RaiseTechSampleAppRouteTableForPublicSubnet
+      ServiceName: com.amazonaws.ap-northeast-1.s3
+      VpcEndpointType: Gateway
+      VpcId: !ImportValue RaiseTechSampleAppVPC

```

こちらもパスカルケースなら下記が適切かなと思いますー！
```suggestion
      VpcId: !ImportValue RaiseTechSampleAppVpc
```

---

修正しました！